apps/preview/next/layouts/Showcases.tsx

```diff
@@ -23,9 +23,18 @@ type GroupsInterface = {
     visible: boolean;
   };
 };
+const inIframe = () => {
+  try {
+    return window.self !== window.top;
+  } catch (e) {
+    return true;
```

Comment:
Why `inIframe` is true in case of an error? 🤔

Reply:
But it feels more like if/else rather than try/catch.

Reply:
I found this solution somewhere, but you are right, I checked now support for `self` and `top` and seems it should work properly everywhere, tested it as well in every browser, here's changes for all suggestions https://github.com/vuestorefront/storefront-ui/pull/2700/commits/a58c7b7131f1092de8182af797a327d8d59b86c6